### PR TITLE
vimc-3230 Added orderly web api url

### PR DESCRIPTION
**Pull request by EmmaLRussell**

### CODE REVIEW

File: settings/dev.json [hunk 1]
```diff
@@ -18,5 +18,6 @@
   "update_on_deploy": false,
   "include_guidance_reports": false,
   "use_production_db_config": false,
-  "copy_static_files": false
+  "copy_static_files": false,
+  "orderly_web_api_url": "http://localhost:8443/reports/api/v1"
```
Comment (hillalex) [inline settings/dev.json]: How come this is different from the suggested value for local dev? https://github.com/vimc/montagu/pull/163/files#diff-7f43a41e5e875b219434f43a6fb8437dR72

Comment (EmmaLRussell) [reply settings/dev.json]: Good question!

File: settings/teamcity.json [hunk 2]
```diff
@@ -18,5 +18,6 @@
     "update_on_deploy": false,
     "copy_static_files": false,
     "include_guidance_reports": false,
-    "use_production_db_config": false
+    "use_production_db_config": false,
+    "orderly_web_api_url": "http://localhost:8443/reports/api/v1"
```
Comment (hillalex) [inline settings/teamcity.json]: This 1 should be the same as dev presumably?

**Comment by hillalex** [comment]
I pointed the api and webapps submodules to the latest commits and the create user tests are failing, but I think just because of the wrong api url in the teamcity settings http://teamcity.montagu.dide.ic.ac.uk:8111/viewLog.html?buildId=78299&tab=buildResultsDiv&buildTypeId=montagu_deploy

File: src/test.py [hunk 3]
```diff
@@ -65,6 +66,55 @@ def work():
...
+            "docker", "run", "--rm",
+            "-v", "demo:/orderly",
+            ow_migrate_image
+        ], check=True)
+
+        ow_cli_image = get_image_name("orderly-web-user-cli", "master")
+        pull(ow_cli_image)
+
+        #user for api blackbox tests
+        run([
```
Comment (hillalex) [inline src/test.py]: Can we pull these repeated blocks out into their own functions, `add_user` and `grant_permissions`?

**Pull request merged**